### PR TITLE
Feature/df druid resto natures vigil

### PR DESCRIPTION
**Pull request by kfinch**

### CODE REVIEW

File: src/analysis/retail/druid/restoration/modules/spells/NaturesVigil.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  }
+
+  onStHeal(event: HealEvent) {
+    if (this.selectedCombatant.hasBuff(TALENTS_DRUID.NATURES_VIGIL_TALENT.id)) {
+      const spellId = event.ability.guid;
+      // if (spellId === SPELLS.REJUVENATION_GERMINATION.id) {
+      //   spellId = SPELLS.REJUVENATION.id;
+      // } else if (spellId === SPELLS.LIFEBLOOM_UNDERGROWTH_HOT_HEAL.id || spellId === SPELLS.LIFEBLOOM_BLOOM_HEAL.id) {
+      //   spellId = SPELLS.LIFEBLOOM_HOT_HEAL.id;
+      // }
```
Comment (emallson) [inline src/analysis/retail/druid/restoration/modules/spells/NaturesVigil.tsx]: stray comments

**Review by emallson — COMMENTED** [review]
stray comments, otherwise looks good

File: src/analysis/retail/druid/restoration/modules/spells/NaturesVigil.tsx (hunk 1 above)
Comment (kfinch) [reply src/analysis/retail/druid/restoration/modules/spells/NaturesVigil.tsx]: oh woops - yeah I was going to coalesce similar spells, then decided it wasn't worth the trouble

**Comment by kfinch** [comment]
nv_breakdown

**Pull request merged**